### PR TITLE
Add PID to the Title

### PR DESCRIPTION
**Pull request by x3rt**
Adds the PID of the SCPSL process to the title.

### CODE REVIEW

**Comment by ALEXWARELLC** [comment]
While I am not a maintainer, I see this could be helpful but as LocalAdmin is designed to hook directly into the server, I'm not sure what @zabszk will say but he's usually full of all sorts of wisdom.

**Comment by zabszk** [comment]
Hi,

LocalAdmin title is changed in more places in the code. You need to change all of them. Otherwise the PID will be removed from the title when Idle Mode is enabled or disabled.

**Comment by x3rt** [comment]
Completely missed that in testing as I have Idle mode disabled.

All works now. Let me know if you don't want me to use expression bodies or any other feedback

**Comment by zabszk** [comment]
The PR looks good now. I will merge it once #76 is merged and conflicts are resolved.

File: Core/LocalAdmin.cs
Comment (ALEXWARELLC) [inline Core/LocalAdmin.cs:792]: Wouldn't a quick check that the process started correctly be good here?

If the process ends up failing to start say, a corruption, how would this be handled?

Comment (zabszk) [reply Core/LocalAdmin.cs:792]: If the process fails to start then `.Start()` throws an exception that is handled by the catch (https://github.com/northwood-studios/LocalAdmin-V2/blob/master/Core/LocalAdmin.cs#L792).

And this is out of the scope of this PR.

**Review by zabszk — APPROVED** [review]

**Comment by zabszk** [comment]
I'm merging this, because I'm still waiting for @MichalPetryka 's opinion about the LF/CRLF in #76.

**Pull request merged**
